src/pages/addCard/components/addCard/AddCardForm.tsx

```diff
@@ -0,0 +1,63 @@
+import { useAddCardForm } from '@pages/addCard/hooks';
+import {
+	CardNumberInput,
+	CardExpiredDateInput,
+	CardHolderNameInput,
+	CardSecurityCodeInput,
+	CardPasswordInput,
+} from '@pages/addCard/components';
```

Comment:
```suggestion
import { useAddCardForm } from './hooks';
import {
	CardNumberInput,
	CardExpiredDateInput,
	CardHolderNameInput,
	CardSecurityCodeInput,
	CardPasswordInput,
} from './components';
```

저는 이렇게 상대경로로 표현되면 의존성의 스코프에 한 눈에 보여서 좋더라고여!
다만 이건 주관적인 시선이라서 정답은 아니랍니다 ㅎㅎ